src/Microsoft.DocAsCode.Build.Engine/Incrementals/DependencyItemSourceInfo.cs

```diff
@@ -70,5 +70,20 @@ public override int GetHashCode()
         {
             return info == null ? null : new DependencyItemSourceInfo(DependencyItemSourceType.File, info);
         }
+
+        public static bool operator ==(DependencyItemSourceInfo a, DependencyItemSourceInfo b)
+        {
+            if (ReferenceEquals(a, b))
```

Comment:
yes, it works

---
In reply to: [104335934](https://github.com/dotnet/docfx/pull/1383#discussion_r104335934) [](ancestors = 104335934)